docs/user-guide/slicing.ipynb

```diff
@@ -123,6 +124,42 @@
     "var"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To avoid subtle and hard-to-spot bugs, positional indexing without dimension label is in general not supported:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    var[1]\n",
+    "except sc.DimensionError as e:\n",
+    "    print(e)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Scipp makes an exception from this rule in the unambiguous case of 1-D objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var1d = var['x', 0]['y', 0]\n",
+    "var1d[1]"
```

Comment:
It might be better to construct a new variable here. The cells shows both slicing with and without dim label and the reader needs to deduce what `var1d` is, which might be unnecessarily complicated.